smart_campus/app/views.py

```diff
@@ -1036,3 +1036,93 @@ def travelplan_delete_page(request, pk):
     travelplan.delete()
 
     return HttpResponseRedirect('/travelplans/')
+
+
+@login_required
+def group_list_page(request):
+    if not request.user.is_administrator():
+        return HttpResponseForbidden()
+
+    group_list = UserGroup.objects.all().order_by('id')
+    paginator = Paginator(group_list, 10)
+
+    page = request.GET.get('page', 1)
+    try:
+        groups = paginator.page(page)
+    except PageNotAnInteger:
+        groups = paginator.page(1)
+    except EmptyPage:
+        groups = paginator.page(paginator.num_pages)
+
+    context = {
+        'email': request.user.email,
+        'categories': StationCategory.objects.all().order_by('id'),
+        'groups': groups
+    }
+
+    return render(request, 'app/group_list_page.html', context)
+
+
+@login_required
+def group_add_page(request):
+    if not request.user.is_administrator():
+        return HttpResponseForbidden()
+
+    if request.method == 'POST':
+        group_name = request.POST.get('name')
+
+        if group_name:
+            try:
+                UserGroup.objects.create(name=group_name)
+                return HttpResponseRedirect('/groups/')
+            except:
```

Comment:
**DO NOT** use `except` or `expect Exception`.
You should declare what exception you gonna catch.

Reply:
Okay.